tests/unit/components/tab/Tab.spec.jsx

```diff
@@ -0,0 +1,37 @@
+import { render, screen, fireEvent } from '@testing-library/react'
+import Tab from '~/components/tab/Tab'
+
+describe('Tab', () => {
+  it('renders the label', () => {
+    const label = 'label text'
+    render(<Tab activeTab label={label} onClick={() => {}} />)
+    const tabLabel = screen.getByText(label)
+    expect(tabLabel).toBeInTheDocument()
+  })
+
+  it('applies activeTab if it is true', () => {
+    const label = 'label text'
+    render(<Tab activeTab label={label} onClick={() => {}} />)
+    const tabButton = screen.getByRole('button')
+    expect(tabButton).not.toHaveClass('activeTab')
+  })
+
+  it('not applies activeTab if it is false', () => {
+    const label = 'label text'
+    render(<Tab activeTab={false} label={label} onClick={() => {}} />)
+    const tabButton = screen.getByRole('button')
+    expect(tabButton).not.toHaveClass('activeTab')
+  })
+
+  it('calls onClick function when clicked', () => {
+    let onClickCalled = false
+    const onClick = () => {
```

Comment:
for mocked functions it is recommended to use mocking function from vitest, i mean `vi.fn()`